tailwind.config.js

```diff
@@ -10,13 +10,19 @@ module.exports = {
       bold: 600
     },
     borderRadius: {
+      none: 0,
       default: '6px',
     },
     boxShadow: {
       small: '0px 2px 4px rgba(0, 0, 0, 0.06)',
       default: '0px 3px 6px rgba(0, 0, 0, 0.06)',
       large: '0px 8px 16px rgba(0, 0, 0, 0.1)',
       xlarge: '0px 15px 30px rgba(0, 0, 0, 0.2)',
+      blue: '0px 2px 2px rgba(0, 110, 255, 0.5), inset 0px -2px 0px rgba(0, 0, 0, 0.1)',
+      'blue-inset': 'inset 0px 2px 0px rgba(0, 0, 0, 0.1)',
+      red: '0px 2px 2px rgba(235, 85, 71, 0.5), inset 0px -2px 0px rgba(0, 0, 0, 0.1)',
+      'red-inset': 'inset 0px 2px 0px rgba(0, 0, 0, 0.2)',
+      none: 'none'
```

Comment:
🤔So I am a bit stumped with this one!

If you check out the designs for the buttons, they have a colored box shadow and inset on hover and active states https://www.figma.com/file/66iSurBAx32DWrJymo9NKL/GCDS-%E2%80%93-Web-UI?node-id=3485%3A1532

These new values `red`, `blue`, `red-inset` and `blue-inset` represent these.

My problem is, they are very specific to the buttons, and feel a bit off to keep them in this file. Any thoughts on this?

Reply:
I think it makes sense to keep them in here since they are still shadows. It'd be nice if we could just control the shadow colour, but its not possible :(